src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/register/AttachedRegisterPage.java

```diff
@@ -0,0 +1,86 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AttachedRegisterPage extends BasePageObject implements RegisterPage {
+
+  @Override public AttachedRegisterPage open() {
+      driver.get(urlBuilder.getUrlForWiki() + URLsContent.SPECIAL_USER_SIGNUP);
+      return this;
+  }
+
+
+  @FindBy(css = "#signupEmail")
+  private WebElement emailField;
+  @FindBy(css = "#signupUsername")
+  private WebElement usernameField;
+  @FindBy(css = "#signupPassword")
+  private WebElement passwordField;
+  @FindBy(css = "#signupBirthDate")
+  private WebElement birthdateField;
+  @FindBy(css = ".birth-month")
+  private WebElement birthMonthField;
+  @FindBy(css = ".birth-day")
+  private WebElement birthDayField;
+  @FindBy(css = ".birth-year")
+  private WebElement birthYearField;
+  @FindBy(css = "#signupSubmit")
+  private WebElement submitButton;
+  @FindBy(css = "#signupForm div:nth-child(2) small")
+  private WebElement usernameError;
+  @FindBy(xpath = "//*[@id=\"signupForm\"]/div[3]/small")
+  private WebElement passwordError;
+  @FindBy(css = "#signupForm > small.error")
+  private WebElement genericError;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)